runhouse/rns/rns_client.py

```diff
@@ -364,8 +364,6 @@ def save_config(self, resource, overwrite: bool = True):
                 f"Resource {rns_address} already exists and overwrite is False."
             )
 
-        config["name"] = rns_address
```

Comment:
In `resource.config_for_rns`, which is called above, this is automatically set. This line is redundant.